plugin/core/windows.py

```diff
@@ -279,6 +281,16 @@ def start_async(self, config: ClientConfig, initiating_view: sublime.View) -> No
             self._new_session = None
             sublime.set_timeout_async(self._dequeue_listener_async)
 
+    def _on_post_session_initialize(
+        self, initiating_view: sublime.View, session: Session, is_error: bool = False
+    ) -> None:
+        if is_error:
+            session.config.erase_view_status(initiating_view)
+            self._new_listener = None
+            self._new_session = None
```

Comment:
All those changes, switching from `on_post_initialize` to a callback due to needing initiating_view to clear the view status.

Maybe we should just not set the status on initializing and not need to clear it then?

Init typically takes a short time, potentially it's not even noticeable.

Reply:
> Maybe we should just not set the status on initializing and not need to clear it then?

I think it's helpful for those times that something initializes very long (e.g. bash-language-server).